### PR TITLE
TRUNK-5898:Add suppose for JSON MIME Type

### PR DESCRIPTION
**Pull request by jwnasambu**
## Description of what I changed
I changed the `getFileMimeType()` to `getFileMimeType(File file)` in the OpenmrsUtil class and made changes to other classes to use the same by swapping the line `String mimeType = getServletContext().getMimeType(f.getName());` to this:
`String mimeType = OpenmrsUtil.getFileMimeType(f);`

Issue I worked on

https://issues.openmrs.org/browse/TRUNK-5898

### CODE REVIEW

**Comment by jwnasambu** [comment]
@ibacher hope I have fixed [this](https://github.com/openmrs/openmrs-core/pull/3543)

File: web/src/main/java/org/openmrs/module/web/ModuleResourcesServlet.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
 		
 		response.setDateHeader("Last-Modified", f.lastModified());
 		response.setContentLength(Long.valueOf(f.length()).intValue());
-		String mimeType = getServletContext().getMimeType(f.getName());
+		String mimeType = OpenmrsUtil.getFileMimeType(f);
```
Comment (ibacher) [inline web/src/main/java/org/openmrs/module/web/ModuleResourcesServlet.java:64]: Let's actually undo this change as well, since it shouldn't be necessary with the change to web.xml.

Comment (jwnasambu) [reply web/src/main/java/org/openmrs/module/web/ModuleResourcesServlet.java]: @ibacher  I have fixed it. Thanks so much.

**Review by ibacher — APPROVED** [review]

**Comment by ibacher** [comment]
LGTM!
